interop/src/build/web/wasm.rs

```diff
@@ -30,23 +31,81 @@ pub(crate) async fn build_wasm(wasm_deploy_path: PathBuf) -> Result<()> {
         spinner.success("Cryptobox ESM bundle [OK]");
     }
 
-    let spinner = RunningProcess::new("Building WASM bundle...", false);
-
-    Command::new("cargo")
-        .args(["make", "wasm"])
-        .current_dir(cwd.join("crypto-ffi"))
-        .stdout(std::process::Stdio::null())
-        .stderr(std::process::Stdio::null())
-        .status()
-        .await?;
-
-    Command::new("bun")
-        .args(["run", "build"])
-        .current_dir(cwd.join("crypto-ffi/bindings/js"))
-        .stdout(std::process::Stdio::null())
-        .stderr(std::process::Stdio::null())
-        .status()
-        .await?;
+    let spinner = RunningProcess::new("Determining whether to build wasm...", false);
+
+    let wasm_modification_time = std::fs::metadata("crypto-ffi/bindings/js/src/autogenerated/core-crypto-ffi_bg.wasm")
+        .and_then(|metadata| metadata.modified())
+        .ok();
+    let last_relevant_mtime = glob("crypto/**/*.rs")?
+        .chain(glob("crypto-ffi/**/*.rs")?)
+        .filter_map(Result::ok)
+        .chain(
+            ["Cargo.lock", "crypto/Cargo.toml", "crypto-ffi/Cargo.toml"]
```

Comment:
If we're going with this, I would suggest to list all relevant workspace members, and all `Cargo.toml` files, as well as `.cargo/*`.